src/main/java/Duke.java

```diff
@@ -1,10 +1,159 @@
+                        "["+tasks[indexing-1].getStatusIcon()+"] "+ list[indexing-1]);
+                System.out.println("____________________________________________________________\n");
+            }
+
+
+            else if(line.toLowerCase().contains("mark"))
+            {
+                indexing=Integer.parseInt(find_index[1]);
+                tasks[indexing-1].markAsDone();
+
+
+
+                skip=true;
+                System.out.println("____________________________________________________________");
+                System.out.println("Nice! I've marked this task as done:\n"+
+                       "["+tasks[indexing-1].getStatusIcon()+"] "+ list[indexing-1]);
+                System.out.println("____________________________________________________________\n");
+
+            }
+
+            else if(line.toLowerCase().contains("todo") || line.toLowerCase().contains("deadline") || line.toLowerCase().contains("event") )
+            {
+
+
+                if(line.toLowerCase().contains("todo"))
+                {
+                    String TodoTask =line.toLowerCase().substring(5,line.length());
+                    list[i]=TodoTask;
+                    tasks[i] = new Todo(TodoTask);
+
+                }
+                else if(line.toLowerCase().contains("deadline")) {
+                    String[] ToSplitDeadline=line.split("/");
+                    String DeadlineTask =ToSplitDeadline[0].toLowerCase().substring(9,ToSplitDeadline[0].length()-1);
+                    list[i]=DeadlineTask;
+                    tasks[i] = new Deadline(DeadlineTask, ToSplitDeadline[1].substring(3,ToSplitDeadline[1].length()));
+                }
+                else if(line.toLowerCase().contains("event"))
+                {
+                    String[] ToSplitEvent=line.split("/");
+                    String EventTask =ToSplitEvent[0].toLowerCase().substring(6,ToSplitEvent[0].length());
+                    list[i]=EventTask;
+                    tasks[i] = new Event(EventTask, ToSplitEvent[1].substring(5,ToSplitEvent[1].length()),ToSplitEvent[2].substring(3,ToSplitEvent[2].length()));
+
+                }
+                skip=true;
+
+                System.out.println("____________________________________________________________");
+                System.out.println("Got it. I've added this task:\n"+
+                                "  "+ tasks[i]+ "\n"+
+                        "Now you have "+ (i+1) +" tasks in the list.");
+
+                System.out.println("____________________________________________________________\n");
+                i+=1;
+
+
+            }
+
+
+            if(line.equalsIgnoreCase("list"))
+            {
+                skip=true;
+                System.out.println("____________________________________________________________\n"+
+                        "Here are the tasks in your list:");
+
+                for(int m=0;m<i;m+=1)
```

Comment:
I recommend using K&R style brackets (aka [Egyptian style]) for your loop statements, as instructed by professor Akshay.
I recommend that you follow the coding convention list 
(https://se-education.org/guides/conventions/java/basic.html) 
and setting your IDE's code style to match what is recommended so as to have an easier time formatting your code.